test/JDBC/input/BABEL-LOGIN-USER-EXT.mix

```diff
@@ -327,6 +327,9 @@ GO
 SELECT name FROM sys.server_principals WHERE principal_id = suser_id('login1    ');
```

Comment:
Add upgrade tests also to verify before and after behaviour verification.

Reply:
Done